Biasing/src/Biasing/TargetBremFilter.cxx

```diff
@@ -101,10 +109,10 @@ void TargetBremFilter::stepping(const G4Step* step) {
       return;
     } else {
       for (auto& secondary_track : *secondaries) {
-        G4String processName =
-            secondary_track->GetCreatorProcess()->GetProcessName();
-
-        if (processName.compareTo("eBrem") == 0 &&
+        auto electron = G4Electron::Definition();
+        auto ebrem_process =
+            simcore::g4user::ptrretrieval::getProcess(electron, "eBrem");
```

Comment:
is it really the electron that's supposed to be here @tomeichlersmith @EinarElen ?

Reply:
I think so... thats the particle that would do the eBrem. Are you asking if this should be a photon whose creator process is eBrem?

Reply:
Yes, that was my thinking, but I guess it works as it is